src/psyclone/dynamo0p3.py

```diff
@@ -760,9 +752,9 @@ def function_space(self):
         are 2 function spaces, return function_space_from. '''
         if self._type == "gh_field":
             return self._function_space1
-        elif self._type in VALID_OPERATOR_NAMES:
+        elif self._type in GH_VALID_OPERATOR_NAMES:
```

Comment:
This can just be `if ...` and same again below.

Reply:
Done.